examples/python/evm/account.swap.py

```diff
@@ -0,0 +1,107 @@
+"""Example showing how to swap tokens using an EVM account."""
+
+import asyncio
+import os
+
+from cdp import Cdp
+from cdp.actions.evm.swap import CreateSwapOptions
+
+
+async def main():
+    """Swap tokens example."""
+    # Initialize CDP SDK
+    api_key_name = os.environ.get("CDP_API_KEY_NAME", "your-api-key-name")
+    api_key_private_key = os.environ.get("CDP_API_KEY_PRIVATE_KEY", "your-api-key-private-key")
+    
+    cdp = Cdp(api_key_name=api_key_name, api_key_private_key=api_key_private_key)
```

Comment:
We use `dotenv` and context manager in all our examples. Would be good to remain consistent here.
See https://github.com/coinbase/cdp-sdk/blob/main/examples/python/evm/account.fund.py#L7 and https://github.com/coinbase/cdp-sdk/blob/main/examples/python/evm/account.fund.py#L11